### ISSUE TITLE
[FR] Possible use of MCP3008 ADC chip to allow use of Mouse/Paddles via the GPIO

### ISSUE THREAD

**Issue by TomDDG** [randyrossi/bmc64#3]
I've seen a few people use these online to convert analogue signals to an input for the Pi. Wondered if it could be used to enhance the joystick ports allowing Mice or Paddles to be used as well as normal digital joysticks (and anything else that uses the analogue input)

http://www.farnell.com/datasheets/808965.pdf

**Comment by randyrossi** [comment]
Yes, it could be done.  I suppose you would only need 4 inputs for POTX/Y for two joysticks so the 3004 would be enough.  It would probably be lower on my list of priorities but would be a neat enhancement.

**Comment by aminch** [comment]
Paddle support was added via the analog input of USB controllers in 3.7: https://github.com/randyrossi/bmc64/blob/master/CHANGELOG.md#37
